cli/internal/helm/client.go

```diff
@@ -7,28 +7,50 @@ SPDX-License-Identifier: AGPL-3.0-only
 package helm
 
 import (
+	"context"
 	"fmt"
+	"log"
+	"strings"
+	"time"
 
+	"github.com/edgelesssys/constellation/v2/internal/config"
+	"github.com/edgelesssys/constellation/v2/internal/constants"
+	"github.com/edgelesssys/constellation/v2/internal/deploy/helm"
+	"github.com/pkg/errors"
 	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
 	"helm.sh/helm/v3/pkg/cli"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+const (
+	// timeout is the maximum time given to the helm client.
+	upgradeTimeout = 1 * time.Minute
```

Comment:
I think a larger timeout could be beneficial here.
Especially when considering upgrades to a large cluster, 1 minute may not be enough.

Maybe we should consider adding `timeout` flags to commands that perform potentially long running operations (Topic for next dev sync?)

Reply:
From looking at the helm code I was under the impression that the timeout is per upgrade action. So cilium, cert-manager, operators and services each have their own 1min timeout. 

Increasing to 3min (?).
Added to dev sync agenda.

Reply:
Added timeout flag, increased default timeout to 3min.